src/main/resources/story/Walmart/walmart-search-product.story

```diff
@@ -0,0 +1,7 @@
+Description: Search for the product.
+
+Scenario: Search for the product.
+Given I am on a page with the URL 'https://www.walmart.com/'
+When I wait until element located `By.xpath(//*[@data-automation-id="header-input-search"])` appears
+When I enter `Froot Loops Mega Size//n` in field located `By.xpath(//*[@data-automation-id="header-input-search"])`
```

Comment:
It is a good practice to put your test values into Examples section

Reply:
Fixed, I used the Examples section, and global variable for the URL.